tests/h/feeds/rss_test.py

```diff
@@ -17,24 +18,31 @@ def _annotation_url():
     return mock.Mock(return_value='https://hypothes.is/a/id')
 
 
+def _annotation(**kwargs):
+    args = dict(
```

Comment:
This seems weirdly inconsistent. Let's have `{'userid': ...}` here not `dict(userid=...)`.